elk/training/ccs.py

```diff
@@ -29,6 +30,7 @@ def __init__(
         loss: Literal["js", "squared"] = "squared",
         num_layers: int = 1,
         pre_ln: bool = False,
+        first_linear_parametrization: Optional[nn.Module] = None,
```

Comment:
I would prefer to have `RecursiveCCS` just directly call `P.register_parametrization` on `.probe[0]` of the appropriate `CCS` object, instead of cluttering `CCS.__init__` with another parameter

Reply:
Done!